src/org/parosproxy/paros/network/ConnectionParam.java

```diff
@@ -86,6 +86,8 @@
 	private static final String PROXY_CHAIN_PROMPT = CONNECTION_BASE_KEY + ".proxyChain.prompt";
 	private static final String TIMEOUT_IN_SECS = CONNECTION_BASE_KEY + ".timeoutInSecs";
 	private static final String SINGLE_COOKIE_REQUEST_HEADER = CONNECTION_BASE_KEY + ".singleCookieRequestHeader";
+	// ZAP: Added option key to enable / disable HTTP State

```

Comment:
Better add the ZAP comment at the top of the class (inline comments are no longer added).